MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -2774,130 +2776,233 @@ The Go Comprehensive API is a project for the future, to follow the C
 Go Utility Functions
 ====================
 
-------------
-Go YDBExit()
-------------
+---------
+Go Exit()
+---------
 
 .. code-block:: go
 
-    YDBExit() int
+    yottadb.Exit() int
 
-If a process wishes to close YottaDB databases and no longer use
-YottaDB, a call to :code:`YDBExit()` wraps `ydb_exit()`_ so that any
-further calls to YottaDB result in a :code:`C.YDB_ERR_CALLINAFTEREXIT`
-error.
+For a process that wishes to close YottaDB databases and no longer use
+YottaDB, the function wraps `ydb_exit()`_ so that any further calls to
+YottaDB result in a :code:`C.YDB_ERR_CALLINAFTEREXIT` error.
 
-Typical processes will not need to call :code:`YDBExit()` because
+Typical processes will not need to call :code:`Exit()` because
 normal process termination closes databases cleanly. However, a
 process that has completed its use of YottaDB, but intends to continue
-with other work for some material period of time, should call
-:code:`YDBExit()`.
+with other work for some non-trivial period of time, should call
+:code:`Exit()`.
 
------------------
-Go YDBForkNCore()
------------------
+See also `Go ForkExec()`_.
+
+-------------
+Go ForkExec()
+-------------
 
 .. code-block:: go
 
-    YDBForkNCore()
+    yottadb.ForkExec(argv0 string, argv []string, attr *ProcAttr) int, int
+
+The function has the same signature as the `Go
+syscall.ForkExec() <https://golang.org/pkg/syscall/#ForkExec>`_, which
+it wraps. The YottaDB `Go ForkExec()`_ ensures that the child process
+is safely disconnected from YottaDB interprocess communication
+resources and database files.
+
```

Comment:
I'm not at all sure that allowing users to specify ProcAttr is a good idea as our version of ForkExec() has a need to set it for YottaDB purposes too which may conflict with the the user. We would end up re-copying the array to add our element to it at best. Also don't like the ability it gives the user by bypass closing all the db files.

Reply:
I don't think it will be acceptable to users to disable it, because ProcAttr includes [SysProcAttr](https://golang.org/src/syscall/exec_linux.go?s=463:1939#L12) some of  which are likely to be important to users. Besides, programs can always explicitly set environment variables using [sys.Setenv()](https://godoc.org/golang.org/x/sys/unix#Setenv). So, I think we should just reserve environment variables with a “ydb_” prefix for our use, and document it (if it is not already).

That said, I wonder whether we should wrap [syscall.StartProcess()](https://golang.org/pkg/syscall/#StartProcess), even though it just [wraps syscall.ForkExec()](https://golang.org/src/syscall/exec_unix.go?s=6712:6811#L230) since syscall.StartProcess() may be the os independent way of starting processes.